fog/view/server/test-utils/src/lib.rs

```diff
@@ -0,0 +1,207 @@
+            router_server: Some(router_server),
+            router_client: Some(router_client),
+            store_servers: Some(store_servers),
+        }
+    }
+
+    /// Creates fog view stores with sane defaults.
+    fn create_view_stores(
+        omap_capacity: u64,
+        store_count: usize,
+        logger: Logger,
+    ) -> (
+        SqlRecoveryDbTestContext,
+        Vec<ViewServer<SgxViewEnclave, AttestClient, SqlRecoveryDb, EpochShardingStrategy>>,
+        Arc<RwLock<HashMap<FogViewStoreUri, Arc<FogViewStoreApiClient>>>>,
+    ) {
+        let db_test_context = SqlRecoveryDbTestContext::new(logger.clone());
+        let mut store_servers = Vec::new();
+        let mut store_clients = HashMap::new();
+
+        for i in 0..store_count {
+            let db = db_test_context.get_db_instance();
+            let (store, store_uri) = {
+                let port = portpicker::pick_unused_port().expect("pick_unused_port");
+                let store_uri = FogViewStoreUri::from_str(&format!(
+                    "insecure-fog-view-store://127.0.0.1:{}",
+                    port
+                ))
+                .unwrap();
+
+                // Each store is responsible for 1 block. Note that this means that the stores
+                // in this test are not responsible for overlapping block ranges.
+                let store_block_range = BlockRange::new(i as u64, (i + 1) as u64);
+                let epoch_sharding_strategy = EpochShardingStrategy::new(store_block_range);
+
+                let config = ViewConfig {
+                    chain_id: "local".to_string(),
+                    client_responder_id: ResponderId::from_str(&store_uri.addr()).unwrap(),
+                    client_listen_uri: Store(store_uri.clone()),
+                    client_auth_token_secret: None,
+                    omap_capacity,
+                    ias_spid: Default::default(),
+                    ias_api_key: Default::default(),
+                    admin_listen_uri: Default::default(),
+                    client_auth_token_max_lifetime: Default::default(),
+                    sharding_strategy: ShardingStrategy::Epoch(epoch_sharding_strategy),
+                    postgres_config: Default::default(),
+                };
+
+                let enclave = SgxViewEnclave::new(
+                    get_enclave_path(mc_fog_view_enclave::ENCLAVE_FILE),
+                    config.client_responder_id.clone(),
+                    config.omap_capacity,
+                    logger.clone(),
+                );
+
+                let ra_client =
+                    AttestClient::new(&config.ias_api_key).expect("Could not create IAS client");
+
+                let mut store = ViewServer::new(
+                    config,
+                    enclave,
+                    db,
+                    ra_client,
+                    SystemTimeProvider::default(),
+                    EpochShardingStrategy::default(),
```

Comment:
:question: Why doesn't this use the `config.sharding_strategy`?

Reply:
Changed.